sally_flower_shop/models/flower.py

```diff
@@ -0,0 +1,20 @@
+from odoo import api, fields, models, tools, _
+from datetime import datetime, timedelta,timezone as timez
+
+class Flower(models.Model):
+    _name = 'flower'
+ 
+    _description = 'A model that describe a flower'
+    
+    common_name = fields.Char()
+    scientific_name = fields.Char()
+    season_start = fields.Date()
+    season_end = fields.Date()
+    watering_frequency = fields.Integer()
+    watering_amount = fields.Float()
```

Comment:
It would be better if you added a string to clarify the unit of the amount e.g. mL or L